scripts/startall.sh

```diff
@@ -0,0 +1,24 @@
+#prepare build
+apt install openjdk-11-jdk telnet bridge-utils maven -y
+git clone https://github.com/futurewei-cloud/alcor.git
+cd alcor
+cp ../Dockerfile scripts/ignite.Dockerfile
```

Comment:
../Dockerfile point to <alcor_root>. Do we refer to <alcor_root>/scripts/Dockerfile? In that case, we will need change the relative path here.